src/components/navbar/Navbar.tsx

```diff
@@ -41,45 +59,56 @@ function Navbar() {
     );
   }
 
+  function verifyMenuItemIcon(itemTitle: string) {
+    switch (itemTitle) {
+      case t("Profile.title"):
+        return <UserOutlined />;
+      case t("Chat.title"):
+        return (
+          <Badge size="small" count={countNotifications}>
+            <BellOutlined />
+          </Badge>
+        );
+      default:
+        return undefined;
+    }
+  }
+
   return (
     <NavbarStyles>
       <NavLink path={navItens?.home ?? ""}>
         <img height="50px" alt="logo" src={logo} />
       </NavLink>
-
-      {screenWidth < MOBILE_SCREEN_SIZE ? (
-        ""
-      ) : (
-        // eslint-disable-next-line react/jsx-no-useless-fragment
-        <>
-          {navItens?.links.map(({ title, to }) => (
-            <NavLink key={to} path={to}>
-              {title}
-            </NavLink>
-          ))}
-        </>
-      )}
-
-      <NavBarButtons>
-        <NavLink path={CHAT_ROUTE}>
-          <NavBarButton icon={<MessageFilled />} title={t("Chat.title")} />
-        </NavLink>
-
-        {navItens?.buttons.map(({ path, title, icon }) => (
-          <NavLink key={title} path={path}>
-            <NavBarButton icon={icon} title={title} />
-          </NavLink>
+      <Menu
+        defaultSelectedKeys={["jobs"]}
+        mode="horizontal"
+        style={{ width: "100%" }}
+      >
+        {role === "client" ? (
```

Comment:
dont compare with string. Move it to const value. But it would be better if u will create object or enum with user roles